Fixed Typographical Error in guide/english/3d/blender/index.md (#35486)
freeCodeCamp/freeCodeCamp: guide/english/3d/blender/index.md

## Blender

Blender is a free and open source 3D creation suite. It's used for modelling, creating photorealistic renders, simulations, model sculpting, architecture models, compositing, character design, animation, rigging, level design, 2D rendered scenes, motion graphics, game development, VFX, video editing, and even much more!

> The Blender software is cross-platform that runs on Windows, Linux, and macOS. Blender is supported by The Blender Foundation, a public-benefit organization from the Netherlands. Developers, animators, scientists, experts and even hobbyists empower the software's development by constantly maintaining, fixing and making new features for suitable and more responsive user experience.

Reviewer: ```suggestion
The Blender software is cross-platform that runs on Windows, Linux, and macOS. Blender is supported by The Blender Foundation, a public-benefit organization from the Netherlands. Developers, animators, scientists, experts and even hobbyists empower the software's development by constantly maintaining, fixing and making new features for a suitable and more responsive user experience.
```

> Blender is now especially useful as the 3D printer technology has become more of a household item for many.  Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files or it can export in a .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to a .stl - a very popular filetype.

Reviewer: ```suggestion
Blender is now especially useful as the 3D printer technology has become more of a household item for many.  Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files ,it can export in the .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to a .stl - a very popular filetype.
```

Reviewer: Please make this change too

> Blender is now especially useful as the 3D printer technology has become more of a household item for many.  Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files, it can export in the .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to a .stl - a very popular filetype.

Reviewer 2: > Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files, it can export in the .stl format as well. 

This isn't really grammatically correct. In fact, I like the previous version as is - 

>  that slicers can convert to .stl files **or** it can export in .stl format as well. 

You don't need a `the` before `.stl format`. Although, you can keep the comma you added, before `or`.

Also, `modeling` isn't incorrect spelling. AFAIK, this is the commonly used spelling in American English. @RandellDawson Thoughts?

Reviewer: modeling is American while modelling is British

Reviewer 3: I believe we typically go with the American spelling here.

The Blender software is cross-platform that runs on Windows, Linux, and macOS. Blender is supported by The Blender Foundation, a public-benefit organization from the Netherlands. Developers, animators, scientists, experts and even hobbyists empower the software's development by constantly maintaining, fixing and making new features for a suitable and more responsive user experience.

> Blender is now especially useful as the 3D printer technology has become more of a household item for many.  Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files, it can export in .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to a .stl - a very popular filetype.

Reviewer: ```suggestion
Blender is now especially useful as 3D printed technology has a household item for many.  Blender allows the user to develop their idea and export it as a .obj that slicers can convert to .stl files, it can export in .stl format as well.  If one develops their project in a CAD app that only exports as .obj, Blender can import the file and convert it to a .stl - a very popular filetype.
```
This looks even better IMO
